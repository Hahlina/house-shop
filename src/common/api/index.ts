export * from './fetchApi';
export * from './services/propety';
