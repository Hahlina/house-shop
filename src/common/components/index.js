export * from './PropertyCard/PropertyCard';
export * from './PropertyCard/PropertyCardLoader';
export * from './SwiperSlide/SwiperSlide';
export * from './SwiperContainer/SwiperContainer';
export * from './NavBreadcrumbs/NavBreadcrumbs';
