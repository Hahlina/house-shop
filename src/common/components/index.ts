export * from './PropertyCard/PropertyCard';
export * from './PropertyCard/PropertyCardLoader';
export * from './SwiperSlide/SwiperSlide';
export * from './SwiperContainer/SwiperContainer';
export * from './NavBreadcrumbs/NavBreadcrumbs';
export * from './Container/Container';
export * from './Image/Image';
export * from './ErrorPage/ErrorPage';
export * from './SomethingWentWrong/SomethingWentWrong';
export * from './NothingFound/NothingFound';
