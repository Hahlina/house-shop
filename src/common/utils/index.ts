export * from './convertPrice';
export * from './capitalizeFirstLetter';
