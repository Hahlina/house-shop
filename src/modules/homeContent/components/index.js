export * from "./PreviewBanner/PreviewBanner";
export * from "./PropertiesList/PropertiesList";
