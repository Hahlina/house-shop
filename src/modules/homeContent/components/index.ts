export * from './PreviewBanner/PreviewBanner';
export * from './PropertiesList/PropertiesList';
export * from './PropertiesPagination/PropertiesPagination';
