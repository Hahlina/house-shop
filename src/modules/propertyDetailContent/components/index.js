export * from "./PropertyGallery/PropertyGallery";
export * from "./FullInformationCard/FullInformationCard";
