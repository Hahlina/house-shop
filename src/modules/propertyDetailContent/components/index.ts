export * from './PropertyGallery/PropertyGallery';
export * from './PropertyGallery/PropertyGalleryLoader';
export * from './FullInformationCard/FullInformationCard';
export * from './FullInformationCard/FullInformationCardLoader';
export * from './PropertyDetailContentLoader/PropertyDetailContentLoader';
export * from './PropertyDetailBreadCrumbs/PropertyDetailBreadCrumbs';
