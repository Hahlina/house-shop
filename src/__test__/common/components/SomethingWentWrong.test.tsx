import { SomethingWentWrong } from 'common/components';

import { render, screen } from 'common/utils/test';

describe('SomethingWentWrong component', () => {
    it('should renders the SomethingWentWrong component with image', () => {
        render(<SomethingWentWrong />);
        const image = screen.getByRole('img');
        expect(image).toBeInTheDocument();
    });

    it('should renders the SomethingWentWrong component with correct alt text', () => {
        render(<SomethingWentWrong />);
        const image = screen.getByAltText(/something went wrong/i);
        expect(image).toBeInTheDocument();
    });

    it('should take a snapshot', () => {
        const { container } = render(<SomethingWentWrong />);
        expect(container).toMatchSnapshot();
    });
});
