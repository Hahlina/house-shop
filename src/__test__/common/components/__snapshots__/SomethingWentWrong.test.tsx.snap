// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SomethingWentWrong component should take a snapshot 1`] = `
<div>
  <div
    class="sc-iBdnpw kymDiO"
  >
    <img
      alt="Something went wrong"
      src="/images/somethingWentWrong.png"
      style="visibility: visible; transition: visibility 200ms ease-out 700ms;"
    />
    ;
  </div>
</div>
`;
